draft-kazuho-httpbis-priority.md

```diff
@@ -94,12 +100,13 @@ interpreted as described in [RFC2119].
 
 # The Priority HTTP Header Field
 
-The Priority HTTP header field is used by a client as part of an HTTP request
-to specify the priority of the response.  A server can also use the header field
-as part of an HTTP request to communicate the correct or amended precedence to
-an intermediary.
+The Priority HTTP header field may appear in requests and responses. A client
```

Comment:
How about s/may/can/? "May" is a RFC2119 word.

Reply:
oops, fixed thanks